doc/README.md

```diff
@@ -0,0 +1,2 @@
+Information: This directory is not a documentation source any more. Developer docs are contained in 
```

Comment:
What about removing the whole directory `doc` (including that new `doc/README.md`)?